community/opendoas/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+printf "/chown \${BINOWN}:\${BINGRP}/d\n/chmod \${BINMODE}/d\nw\nq" |
+    ed bsd.prog.mk >/dev/null
+
+./configure \
+    --enable-static \
+    --without-pam
```

Comment:
What functionality is lost when `pam` is disabled (if any)?

Reply:
nothing that matters in a KISS system. `shadow` is still enabled by default.